kenmerkendewaarden/overschrijding.py

```diff
@@ -339,6 +341,7 @@ def cost_func(params, *args):
         x0=initial_guess,
         args=(p_val_gt_threshold, threshold, values[values > threshold]),
         method="Nelder-Mead",
+        bounds = [[-math.inf, math.inf], [1e-10, math.inf]],
```

Comment:
I think `math.inf` can be replaced with `None` which might be slightly neater since it reverts to the default bounds in that case (which are most probably +/- `math.inf`)

Reply:
I tried a few thinks like you suggested, but ended up with errors as: 'NoneType' object is not iterable, so I keep it as it is.

Reply:
Maybe I should have been more clear, my suggestion was to use `bounds = [[None, None], [1e-10, None]],`. I think I successfully ran the testbank with this, but you might have done another test?